packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -17,10 +17,12 @@ import {
   withStyles,
   PolymorphicForwardRef,
   toPascalCase,
+  toCamelCase,
 } from "../../utils";
 import styles from "./Box.module.scss";
+import textStyles from "../Text/Text.module.scss";
 
-type spacing = "xs" | "s" | "m" | "l" | "xl";
+type spacing = "xs" | "s" | "m" | "l" | "xl" | "0" | false;
```

Comment:
zero string type! Could be confusing but I'm not going to worry about that for now since that is the API exposed in the DOM